src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 600;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 20px 7.6px 7.5px;
```

Comment:
why do you need this decimal values?

Reply:
Because i will fail test without this decimal values.